src/components/Match/matchColumns.jsx

```diff
@@ -706,3 +660,132 @@ export const teamfightColumns = [
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/ward_sentry_lg.png`} role="presentation" />
+        &nbsp;{strings.th_use_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_used_ward_sentry,
+    sortFn: true,
+    displayFn: row => row.item_uses.ward_sentry || '-',
+  }, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/dust_lg.png`} role="presentation" />
+        &nbsp;{strings.th_purchase_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_purchase_dust,
+    field: 'purchase_dust',
+    sortFn: true,
+    displayFn: (row, col, field) => field || '-',
+  }, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/dust_lg.png`} role="presentation" />
+        &nbsp;{strings.th_use_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_used_dust,
+    sortFn: true,
+    displayFn: row => row.item_uses.dust || '-',
+  }, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/smoke_of_deceit_lg.png`} role="presentation" />
+        &nbsp;{strings.th_purchase_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_purchase_smoke_of_deceit,
+    field: 'purchase_smoke_of_deceit',
+    sortFn: true,
+    displayFn: (row, col, field) => field || '-',
+  }, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/smoke_of_deceit_lg.png`} role="presentation" />
+        &nbsp;{strings.th_use_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_used_smoke_of_deceit,
+    sortFn: true,
+    displayFn: row => row.item_uses.smoke_of_deceit || '-',
+  }, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/gem_lg.png`} role="presentation" />
+      </div>
+    ),
+    tooltip: strings.tooltip_purchase_gem,
+    field: 'purchase_gem',
+    sortFn: true,
+    displayFn: (row, col, field) => field || '-',
+  },
+];
+
+export const supportColumns = [
+  heroTdColumn, {
+    center: true,
+    displayName: strings.th_stacked,
+    tooltip: strings.tooltip_camps_stacked,
+    field: 'camps_stacked',
+    sortFn: true,
+    displayFn: (row, col, field) => field || '-',
+  }, {
+    center: true,
+    displayName: strings.th_tpscroll,
+    tooltip: strings.tooltip_purchase_tpscroll,
+    field: 'purchase_tpscroll',
+    sortFn: true,
+    displayFn: (row, col, field) => field || '-',
+  },
+  visionColumns[1],
```

Comment:
you might want to define these columns as individual variables and reference by name in both places.  Trying to track down specific indexes in the array is more difficult and prone to breakage if someone changes the order or something.